src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -99,12 +100,12 @@ static PSVersionInfo()
                 rawGitCommitId = mainVersion;
             }
 
-            s_psV6Version = new SemanticVersion(mainVersion);
+            s_psV7Version = new SemanticVersion(mainVersion);
```

Comment:
Will we rename for every main version? Can we use unified name?

Reply:
This is needed as there is an array of all compatible PS versions